### PR TITLE
feat: add shap-value modal

### PR DESCRIPTION
**Pull request by Jeffrey-Zutt**
Adds the first version of a shap-value details modal.

### CODE REVIEW

File: src/components/cases/FraudPrediction/utils/parseShapValues.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+   'ratio_gezinsverhouding_6': 'Ratio alleenstaande'
+}
+
+const translate = (str:string) => translationMap[str] ?? str
+
+const humanize = (str:string) => {
+  // Replace underscore with ' '
+  const replaced = str.replace(/_/g, ' ')
+  // Capitalize
+  return replaced.charAt(0).toUpperCase() + replaced.slice(1)
```
Comment (gilleswittenberg) [inline src/components/cases/FraudPrediction/utils/parseShapValues.ts:88]: There is also `src/lib/utils/capitalize.ts`

File: src/types/api.d.ts [hunk 2]
```diff
@@ -29,9 +29,13 @@ declare type PostalCode = string
 declare type Stadium = string
 declare type Stadia = Stadium[]
 declare type Distance = number
+declare type ShapValues = { [key:string]:number }
+declare type BusinessRules = { [key:string]:number }
```
Comment (gilleswittenberg) [inline src/types/api.d.ts:33]: `Record<string, number>` would be cleaner imo

Comment (Jeffrey-Zutt) [reply src/types/api.d.ts:33]: Nice! didn't know about that one yet.

File: src/components/cases/FraudPrediction/utils/parseShapValues.ts (hunk 1 above)
Comment (Jeffrey-Zutt) [reply src/components/cases/FraudPrediction/utils/parseShapValues.ts:88]: Awesome. Will change

**Pull request merged**